cal-itp/benefits
# Refactor: remove load sample data

Closes #1243 

This PR removes the `LOAD_SAMPLE_DATA` setting, the `DJANGO_MIGRATION_DIR` setting, and all the code / documentation that were related to them.

This means that all instances of the app (local and actual deployments) will use the `0002_data.py` migration file to populate its database. Configuration can be customized by setting environment variables.

## To test
There are two use cases that should still work - namely, the use case of running with sample data and of running with custom configuration.

### Run with sample data
1. With no environment variables set, run `./bin/init.sh`
1. Check that the database contains sample values by running `code django.db`

### Run with custom configuration
1. Choose some environment variables to set, and run `./bin/init.sh` again to rebuild the database
1. Check that the database contains the values that you overrode via environment variables

Also, in both cases, check that the app runs properly.
